x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/plugin/RestSqlQueryAction.java

```diff
@@ -124,6 +128,11 @@ public RestResponse buildResponse(SqlQueryResponse response) throws Exception {
         });
     }
 
+    @Override
+    protected Set<String> responseParams() {
+        return textFormat == TextFormat.CSV ? Collections.singleton("delimiter") : Collections.emptySet();
```

Comment:
Can we use some static variable for "delimiter"?

Reply:
Sure. I've updated along a few other embedded strings, for this change to make sense.